ansible/playbook.yml

```diff
@@ -0,0 +1,16 @@
+---
+- hosts: all
+  become: True
+  gather_facts: False
+  roles:
+    - base
+    - openjdk
+    - fuseki
```

Comment:
Fuseki is installed but not actually used by Skosmos here (it is configured to use the Finto dev SPARQL endpoint). I think it would be better to start with a Skosmos-only Vagrant install. Later we can change to include Fuseki as well, with some vocabularies loaded and actually used by Skosmos